core/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -963,8 +963,8 @@ public boolean canMatch(ShardSearchRequest request) throws IOException {
     }
 
     /**
-     * Returns true iff the given search source builder can be early terminated by rewriting to a match none query. Or in other words
-     * if the execution of a the search request can be early terminated without executing it. This is for instance not possible if
```

Comment:
Looks like you just changed both the first occurance of "iff" and the later "if" to the other version. Could you just change them back?